modules/bullet/bullet_physics_server.h

```diff
@@ -51,44 +51,61 @@ class BulletPhysicsServer3D : public PhysicsServer3D {
 	friend class BulletPhysicsDirectSpaceState;
 
 	bool active = true;
-	char active_spaces_count = 0;
-	Vector<SpaceBullet *> active_spaces;
+	bool using_threads = false;
+	bool doing_sync;
```

Comment:
Should be initialized in the header (and then doesn't need to be initialized in constructor, unless there's custom logic to set it).

Reply:
![image](https://user-images.githubusercontent.com/76991284/115796280-dd3a2600-a3d1-11eb-9f37-b6b2120a9485.png)
Then have fun with the default physics classes.. every where the same think.

Reply:
As you noticed, lots of classes don't follow the coding style, but they will be updated over time. The point is not to add more code that doesn't follow the coding style.

Reply:
Okai. Got it. In the new commit i was fixing your requested changes.